packages/terra-date-time-picker/src/terra-dev-site/common/DateTimePickerTimezoneExampleTemplate.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import DateTimePicker from 'terra-date-time-picker/lib/DateTimePicker';
+import DateTimeUtils from '../../DateTimeUtils';
+
+const propTypes = {
+  /**
+   * The current entered date time. Use for the selected date message.
+   */
+  value: PropTypes.string,
+  /**
+   * Timezone for the provided date time.
+   */
+  initialTimeZone: PropTypes.string,
+};
+
+const defaultProps = {
+  value: '',
+  initialTimeZone: DateTimeUtils.getLocalTimeZone(),
+};
+
+function DateTimePickerTimezoneTemplate(props) {
+  let dateTimeDisplay = props.value;
+  let timeZoneDisplay = props.initialTimeZone;
+
+  const computedDateTime = DateTimeUtils.createSafeDate(dateTimeDisplay, props.initialTimeZone);
+  if (computedDateTime && computedDateTime.isValid()) {
+    dateTimeDisplay = computedDateTime.format();
+    timeZoneDisplay = computedDateTime.tz();
+  }
+  const [dateTime, setDateTime] = useState(dateTimeDisplay);
+  const timeZone = timeZoneDisplay;
+
+  const handleDateTimeChange = (event, time) => {
+    setDateTime(time);
+  };
+
+  return (
+    <>
+      <p>
+        Selected ISO Date Time:
+        <span id="date-time-value">{dateTime}</span>
+      </p>
+      <p>
+        Selected Time Zone:
```

Comment:
This should say something like `Initial Time Zone` instead of `Selected Time Zone` to avoid thinking that it can be changed on the fly.

Reply:
changed here d09c6b6938c6c9fe12cf082e09fd4ce65bd55da6